documentation/migration-guide.md

```diff
@@ -0,0 +1,200 @@
+# Migration Guide
+
+This guide helps you migrate your ReadMe CLI (`rdme`) setup to the latest version and prepare for future versions. Choose your migration path based on whether you're using [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored).
+
+1. If your project is using [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored), [use `rdme@v10`](#migrating-to-rdme10) and beyond.
+2. If your project is **not** yet using ReadMe Refactored, [use `rdme@v9`](#migrating-to-rdme9). The `v9` channel will continue to be maintained while we focus on making sure that everybody can upgrade their ReadMe projects to ReadMe Refactored.
+
+## Table of Contents
+
+<!--
+
+Autogenerated TOC — to regenerate:
+
+npx markdown-toc documentation/migration-guide.md --maxdepth 2 --bullets="-" -i
+
+ -->
+
+<!-- toc -->
+
+- [Migrating to `rdme@10`](#migrating-to-rdme10)
+- [Migrating to `rdme@9`](#migrating-to-rdme9)
+- [Migrating to `rdme@8`](#migrating-to-rdme8)
+- [Version Compatibility Matrix](#version-compatibility-matrix)
+- [Need Help?](#need-help)
+
+<!-- tocstop -->
+
+## Migrating to `rdme@10`
+
+<!-- prettier-ignore-start -->
+> [!NOTE]
+> `rdme@10` has not been released yet. If you're using ReadMe Refactored, stay tuned!
+<!-- prettier-ignore-end -->
```

Comment:
prettier bby what are you doing

![Screenshot 2024-12-05 at 10 25 10 AM](https://github.com/user-attachments/assets/d02af428-7010-4b3c-b33f-5375e792be30)